macros/stitch/base/stitch_fb_ad_creatives__child_links.sql

```diff
@@ -0,0 +1,101 @@
+-- There are cases where ads can have multiple links attached to them.
```

Comment:
The core logic that you're trying to do across redshift and snowflake is identical here, it's just that Stitch loads the data in differently. My recommendation would be to create a base model that normalizes the schemas across both Redshift and snowflake, and then do the transformation in a single model that doesn't need two different adapters because the underlying data is identical. We can accomplish what you're doing here much more simply and minimize the maintenance burden for future analysts.